packages/catalog-model/src/location/validation.ts

```diff
@@ -21,6 +21,7 @@ export const locationSpecSchema = yup
   .object<LocationSpec>({
     type: yup.string().required(),
     target: yup.string().required(),
+    pendingLocation: yup.bool(),
```

Comment:
Done.

Reply:
Changed to `presence: yup.string()` because adding either `.notRequired()` or `.oneOf(['optional', 'required'])` was causing Typescript errors I couldn't sort out.